piet-common/src/cairo_back.rs

```diff
@@ -163,34 +163,38 @@ impl<'a> BitmapTarget<'a> {
         Ok(size)
     }
 
-    /// Get raw RGBA pixels from the bitmap.
-    pub fn raw_pixels(&mut self, fmt: ImageFormat) -> Result<Vec<u8>, piet::Error> {
+    /// Get an in-memory pixel buffer from the bitmap.
+    pub fn make_image_buf(&mut self, fmt: ImageFormat) -> Result<ImageBuf, piet::Error> {
```

Comment:
I think by ]rust naming conventions](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv) this can be `to_image_buf`?

Reply:
That was my first attempt, but clippy complained about a `to_xxx` function taking `&mut self`...

Reply:
ah gotcha. hmhm...

Reply:
options:

- this name
- `#[ignore(clippy::XXX)]`
- interior mutability

I'm personally okay with option 2. I think the confusion of having this method requrie `&mut self` is lower than the confusion of a new prefix.

I think option 3 is also something to discuss, because these signatures do "feel" wrong, but that's a more invasive change.

Option 1 is also reasonable.

I'm not feeling very decisive; you should feel free to go with whatever you think makes sense.

Reply:
I think I like option 2 best.